<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>smallnote</class>
 <widget class="QWidget" name="smallnote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmallNote</string>
  </property>
  <widget class="QLabel" name="Text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>内容</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextIn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="OK">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>143</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="NO">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>143</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <action name="actionOK">
   <property name="text">
    <string>OK</string>
   </property>
  </action>
  <action name="actionNO">
   <property name="text">
    <string>NO</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
